scripts/ock-once-swtok.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+PIN=${OCK_PIN:-"12345678"}
+
+if id -u | grep -qw 0 && groups | grep -qwv pkcs11 ; then
+    echo "missing pkcs11 group membership"
+    exit 77
+fi
+
+if ! command -v pkcsslotd > /dev/null; then
+    echo "pkcsslotd is required"
+    exit 77
+fi
+
+for PREFIX in "/usr/local/" "/" "NA"; do
+    if [ "$PREFIX" = "NA" ]; then
+        echo "No ock configuration found or ock-swtok not configured"
+        exit 77
+    fi
+
+    OCK_CONF=${PREFIX}etc/opencryptoki/opencryptoki.conf
```

Comment:
path relative to test dir, no system paths please

Reply:
AFAIK, this is currently not possible with opencryptoki.

Reply:
then I guess we'll need to hold on this PR until this is addressed.